pkg/service/test/match.go

```diff
@@ -72,7 +87,11 @@ func jsonMatch(key string, expected, actual interface{}, noiseMap map[string]boo
 	}
 	switch x.Kind() {
 	case reflect.Float64, reflect.String, reflect.Bool:
-		if expected != actual && !noiseMap[key] {
+		regexArr, ok := CheckStringExist(key, noiseMap)
+		if ok && len(regexArr) != 0 {
```

Comment:
instead of ok name it as isNoise or something like that

Reply:
done!